fine_tuning_mistral_beavertails/beavertails.ipynb

```diff
@@ -0,0 +1,5831 @@
+      ],
+      "metadata": {
+        "id": "kvkKe8K7Oqq9"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## How to improve Dataset Credibility\n",
```

Comment:
what is "dataset credibility"? probably want to define something like that before introducing. I would think it means something along the lines of the accuracy of the data, which doesn't make sense in terms of PII / toxicity / sentiment / etc. (none of these are inaccuracies)

Reply:
I will add a section on "What is Dataset Credibility?". But from the outlook, I see the term as not just accuracy related (i know where you are coming at, when you say accuracy!). Using a dataset to train a model is also about banking on it's reliability, source it has come from, and more importantly in AI safety - trustworthyness. 

Imagine if my fine-tuned LLM gives out profane responses, just like the many other LLM incidents out there.